app/src/main/java/com/asapp/churros/rx/Extensions.kt

```diff
@@ -12,6 +12,7 @@ fun Disposable.disposeWith(disposableContainer: DisposableContainer) {
     disposableContainer.add(this)
 }
 
+@Deprecated("Use the simpler com.asapp.churros.runIf. Will be removed in the next mayor version.")
```

Comment:
minor typo (major) :)

```suggestion
@Deprecated("Use the simpler com.asapp.churros.runIf. Will be removed in the next major version.")
```